src/main/java/seedu/duke/data/LessonManager.java

```diff
@@ -0,0 +1,126 @@
+package seedu.duke.data;
+
+import seedu.duke.exception.LessonInvalidTimeException;
+import seedu.duke.exception.LessonNotFoundException;
+
+import java.time.DayOfWeek;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class LessonManager {
+    private static final HashMap<DayOfWeek, ArrayList<Lesson>> lessonMap = new HashMap<>();
```

Comment:
Now the manager is able to take values from Mon to Fri. Will attempt to update it to be more date specific.